code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1318,3 +1318,21 @@
 	to_chat(src, span_danger("You shove [target.name] into [name]!"))
 	log_combat(src, target, "shoved", "into [name]")
 	return COMSIG_CARBON_SHOVE_HANDLED
+
+
+// Checks to see how many hands this person has to sign with.
+/mob/living/carbon/proc/check_signables_state()
+	var/mob/living/carbon/mute = src
```

Comment:
why do you cast src to a carbon when it's already a carbon?

Reply:
overlooked that, I'll fix